components/CustomInput/CustomInput.module.scss

```diff
@@ -1,11 +1,14 @@
 @import 'colors';
-.customInput {
+
+.custom-input {
   margin-bottom: 23px;
 }
-.labelInput {
-  font-size: 30px;
+
+.label-input {
   color: $gray-850;
+  font-size: 30px;
 }
```

Comment:
restore to camelCase and implement `.text-3xl` from styles\fonts.scss

Reply:
i change for this 
![image](https://user-images.githubusercontent.com/10248108/189452967-b4c2a439-f36d-45b0-8cd8-265a10aba725.png)